lighthouse-core/gather/driver.js

```diff
@@ -150,7 +150,11 @@ class Driver {
         returnByValue: true
       }).then(result => {
         clearTimeout(asyncTimeout);
-        resolve(result.result.value);
```

Comment:
hmm, this is actually a bit worrying. We check for errors differently in the CLI and the extension :S The CLI looks for `result.error`, the extension [looks at `result.exceptionDetails`](https://github.com/GoogleChrome/lighthouse/blob/0220f269a23bb20917b0a5b4e6dfedf0d5fb95ca/lighthouse-core/gather/connections/extension.js#L120) as you are here (actually it looks for `result.wasThrown` and then reports `exceptionDetails` if so).

is `wasThrown` also set on this object? We may need to get both `connection.handleRawMessage` and `extension.sendCommand` doing the same thing here.